config/environments/production.rb

```diff
@@ -19,8 +19,7 @@
   config.assets.compress = true
 
   # Fallback to the asset pipeline if a precompiled asset is missed.
-  # TODO: Once everything is in the asset pipeline, this should be set back to false.
-  config.assets.compile = true
+  config.assets.compile = false
```

Comment:
🎉 Yay!  🌮 

This always felt weird to maintain two types of assets, ones served through the pipeline and others that were not.

Reply:
IMO we're serving assets through apache and `conf.public_file_server.enabled` is default to `false`.